### DISCUSSION TITLE
Read transactions

### DISCUSSION THREAD

**Discussion by venkat-murty** [erthink/libmdbx#332] [Q&A] [answered]
Can read transactions be shared across multiple threads simultaneously ?
Ex:
Thread 0:
   txn = open a READ ONLY transactions.
   spawn threads 1 and 2 (passing txn).
  join threads 1 and 2.
  close txn.

Thread 1
   read using txn

Thread 2:
   read from txn

**Comment by erthink** [comment, marked as answer]
Please RTFM about the [`MDBX_NOSTICKYTHREADS`](https://libmdbx.dqdkfa.ru/group__c__opening.html#gga9138119a904355d245777c4119534061a9e8785fc7a961c5671dc046b4322647e).
However, you should not perform any actions with the same transaction object simultaneously from different threads, including open/close a cursors, etc.
